src/experimental/safe_html_transform/browser/controlpanel.py

```diff
@@ -0,0 +1,153 @@
+from Products.CMFPlone import PloneMessageFactory as _
+from plone.app.registry.browser import controlpanel
+# from Products.CMFPlone.interfaces import IFilterSchema
+from Products.CMFCore.interfaces import ISiteRoot
+from plone.app.layout.navigation.interfaces import INavigationRoot
+# from plone.registry.interfaces import IRegistry
+from zope.interface import Interface
+# from zope.component import adapts
+from zope import schema
+# from zope.component import getUtility
```

Comment:
If you don't need them, delete them, don't comment them out. You can always find them again with the git history. Nothing is lost forever :)